aim/agent/aid/universes/aci/converter.py

```diff
@@ -202,6 +202,196 @@ def fv_rs_path_att_converter(object_dict, otype, helper,
     return result
 
 
+def qos_dpp_pol_converter(object_dict, otype, helper,
+                          source_identity_attributes,
+                          destination_identity_attributes, to_aim=True):
+    result = []
+    if to_aim:
+        res_dict = {}
+        try:
+            id = default_identity_converter(object_dict, otype, helper,
+                                            to_aim=True)
+        except apic_client.DNManager.InvalidNameFormat:
+            return []
+        for index, attr in enumerate(destination_identity_attributes):
+            res_dict[attr] = id[index]
+        for key in ['rate', 'burst']:
+            if object_dict.get(key):
+                res_dict[key] = object_dict.get(key)
+        if object_dict.get('displayName'):
+            res_dict['display_name'] = object_dict.get('displayName')
+        result.append(default_to_resource(res_dict, helper, to_aim=True))
+    else:
+        dn = default_identity_converter(object_dict, otype, helper,
+                                        aci_mo_type=helper['resource'],
+                                        to_aim=False)[0]
+        res_dict = {'dn': dn}
+        if object_dict.get('rate'):
+            res_dict['rate'] = object_dict.get('rate')
+            res_dict['rateUnit'] = 'kilo'
```

Comment:
As suggested above, we probably should expose the rateUnit, along with other attributes, as part of the AIM resource, rather than hard-code this. Any hard-coding needed could be handled in the AIM consumer (e.g. OpenStack GBP unified plugin). I think this will also make the converters simpler - you may be able to use default ones with some extra properties in the mapping.

Reply:
Done